crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -46,7 +46,13 @@ def knife_files
       crowbar_databags_path.children.each do |file|
         file_path = crowbar_databags_path.join(file)
 
-        if file.basename.to_s =~ /^bc-nova_dashboard-(.*)\.json$/
+        case file.basename.to_s
+        when /^bc-crowbar-(.*)\.json$/
+          attributes = attribute.split(".")
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `attributes`. Did you mean `attribute`? (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)